uploadservice-s3/src/main/java/net/gotev/uploadservice/s3/S3Setup.kt

```diff
@@ -0,0 +1,48 @@
+package net.gotev.uploadservice.s3
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import com.amazonaws.mobileconnectors.s3.transferutility.TransferService
+import java.util.UUID
+
+class S3Setup(context: Context) {
+    private val context = context
+
+    /**
+     * You should call this once. It can be either on activity create or the first time you want to do the upload.
+     */
+    fun startTransferService() {
```

Comment:
why is this not tracked via a private variable? `bool isStarted?` if (!isStarted) { // start } else { log('already started')`